extensions/gson/src/test/groovy/io/jsonwebtoken/gson/io/GsonDeserializerTest.groovy

```diff
@@ -17,14 +17,22 @@ package io.jsonwebtoken.gson.io
 
 import com.google.gson.Gson
 import io.jsonwebtoken.io.DeserializationException
+import io.jsonwebtoken.io.Deserializer
 import io.jsonwebtoken.lang.Strings
+import org.hamcrest.CoreMatchers
```

Comment:
we use EasyMock's API everywhere else in the project - can we please stick with those?